### DISCUSSION TITLE
Project Proposal

### DISCUSSION THREAD

**Discussion by fstayco** [Cirrolytix/gerry_mentorship#20] [General]
Hi @Gerpap24 .  Let's document our conversations and knowledge sharing here.  

Regarding your question:

![image](https://github.com/Cirrolytix/gerry_mentorship/assets/56645234/1a826ea6-87e3-485f-a9c2-4c3abc5a3919)
![image](https://github.com/Cirrolytix/gerry_mentorship/assets/56645234/16b68788-fe89-43a2-8958-94a62a914c1c)

I recorded a prompting guide to help you.  Please build on what I have created to polish your proposal.  
[https://drive.google.com/file/d/15BsWyhQZtAamqlygsMMkFZT96hRInRrL/view?usp=drive_link](https://drive.google.com/file/d/15BsWyhQZtAamqlygsMMkFZT96hRInRrL/view?usp=drive_link)

**Comment by Gerpap24** [comment]
Quite challenging maam. Need a lot of heart.
Anyways,  I know now how to maximize Copilots' potential -have a good detailed prompt.
